### PR TITLE
DOCSP-44953 TOC Relabel

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-spark-connector/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-44953>
Staging - <https://deploy-preview-214--docs-spark-connector.netlify.app/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/batch-mode.txt [hunk 1]
```diff
@@ -10,8 +10,8 @@ Batch Mode
 
 .. toctree::
 
-   /batch-mode/batch-read
-   /batch-mode/batch-write
+   Read in Batch Mode </batch-mode/batch-read>
```
Comment (mongoKart) [inline source/batch-mode.txt:13]: Could just say "Read" and "Write"?

File: source/batch-mode/batch-read.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ Read from MongoDB in Batch Mode
 .. toctree::
    :caption: Batch Read Configuration Options
 
-   /batch-mode/batch-read-config
+   Configuration Options </batch-mode/batch-read-config>
```
Comment (mongoKart) [inline source/batch-mode/batch-read.txt:10]:
```suggestion
   Configuration </batch-mode/batch-read-config>
```

File: source/index.txt [hunk 3]
```diff
@@ -2,6 +2,18 @@
 MongoDB Connector for Spark
 ===========================
 
+.. toctree::
+   :titlesonly:
+
+   Get Started <getting-started>
+   Configure Spark <configuration>
+   Configure TLS/SSL <tls>
```
Comment (mongoKart) [inline source/index.txt:10]: To avoid repetition with previous entry

```suggestion
   TLS/SSL <tls>
```

File: source/streaming-mode.txt [hunk 4]
```diff
@@ -12,8 +12,8 @@ Streaming Mode
 
 .. toctree::
 
-   /streaming-mode/streaming-read
-   /streaming-mode/streaming-write
+   Read in Streaming Mode </streaming-mode/streaming-read>
```
Comment (mongoKart) [inline source/streaming-mode.txt:15]: Read / Write only

File: source/streaming-mode/streaming-read.txt [hunk 5]
```diff
@@ -7,7 +7,7 @@ Read from MongoDB in Streaming Mode
 .. toctree::
    :caption: Streaming Read Configuration Options
 
-   /streaming-mode/streaming-read-config
+   Configuration Options </streaming-mode/streaming-read-config>
```
Comment (mongoKart) [inline source/streaming-mode/streaming-read.txt:10]:
```suggestion
   Configuration </streaming-mode/streaming-read-config>
```

File: source/streaming-mode/streaming-write.txt [hunk 6]
```diff
@@ -7,7 +7,7 @@ Write to MongoDB in Streaming Mode
 .. toctree::
    :caption: Streaming Write Configuration Options
 
-   /streaming-mode/streaming-write-config
+   Configuration Options </streaming-mode/streaming-write-config>
```
Comment (mongoKart) [inline source/streaming-mode/streaming-write.txt:10]:
```suggestion
   Configuration </streaming-mode/streaming-write-config>
```

**Review by mongoKart — APPROVED** [review]
LGTM. I made some suggestions, but do what you think makes sense!

File: source/streaming-mode.txt (hunk 4 above)
Comment (sarahemlin) [reply source/streaming-mode.txt:15]: Same thoughts

File: source/index.txt (hunk 3 above)
Comment (lindseymoore) [reply source/index.txt]: Left as is, because it feels wrong to leave out the word on the top level w/ no context otherwise

**Pull request merged**